hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/util/S3Consts.java

```diff
@@ -64,4 +64,9 @@ private S3Consts() {
 
   public static final String CUSTOM_METADATA_HEADER_PREFIX = "x-amz-meta-";
 
+  public static final String ENDPOINT_STYLE_PARAM = "endpoint-style";
+
+  public static final String ENDPOINT_STYLE_PATH = "path";
+
+  public static final String ENDPOINT_STYLE_VIRTUAL = "virtual";
```

Comment:
Where did these values come from? I can't find them documented by AWS anywhere, for example at https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html

Reply:
I added these internally to identify when virtual host style addressing is used.